src/NServiceBus.Persistence.DynamoDB/Saga/SagaPersister.cs

```diff
@@ -102,7 +232,9 @@ public Task Update(IContainSagaData sagaData, ISynchronizedStorageSession sessio
             map.Add(configuration.PartitionKeyName, new AttributeValue { S = $"SAGA#{sagaData.Id}" });
             map.Add(configuration.SortKeyName, new AttributeValue { S = $"SAGA#{sagaData.Id}" });  //Sort key
             // Version should probably be properly moved into metadata to not clash with existing things
-            map.Add("___VERSION___", new AttributeValue { N = version.ToString() });
+            map.Add(SagaDataVersionAttributeName, new AttributeValue { N = version.ToString() });
+            // release lock on save
+            map.Add(SagaLockAttributeName, new AttributeValue { N = "-1" });
```

Comment:
maybe we could investgiate in approaches to delete the column instead of setting it to -1